base/src/main/scala/co/uproot/abandon/BuildInfo.scala

```diff
@@ -0,0 +1,20 @@
+package co.uproot.abandon
+
+import scala.Predef._
+
+/** This object was generated by sbt-buildinfo. */
```

Comment:
I'm a bit confused. How are we going to reference the file when it is not being tracked by Git?